src/tests/gobprepare/test_prepare_client.py

```diff
@@ -516,6 +516,95 @@ def test_run_prepare_action_check_row_counts(self, mock_logger):
         self.assertFalse(result["row_count_check"])
         mock_logger.error.assert_called_with("Deviation of -20.00% for growing_table! Expected 500 rows, got 400")
 
+    def test_run_prepare_action_check_source_sync_complete(self, mock_logger):
+        prepare_client = PrepareClient(self.mock_dataset, self.mock_msg)
+        prepare_client.action_check_source_sync_complete = MagicMock(return_value=True)
+        action = {
+            "type": "check_source_sync_complete",
+            "id": "check_source_sync",
+            'table_name': 'some_table_name',
+            'schema': 'some_schema'
+        }
+
+        result = prepare_client._run_prepare_action(action)
+        prepare_client.action_check_source_sync_complete.assert_called_with(action)
+        self.assertTrue(result['check_source_sync'])
+
+    def test_run_prepare_action_complete_prepare(self, mock_logger):
+            prepare_client = PrepareClient(self.mock_dataset, self.mock_msg)
+            prepare_client.action_complete_prepare = MagicMock(return_value=True)
+            action = {
+                "type": "complete_prepare",
+                "id": "update_after_prepare_complete",
+                'table_name': 'some_table_name',
+                'schema': 'some_schema'
+            }
+
+            result = prepare_client._run_prepare_action(action)
+            prepare_client.action_complete_prepare.assert_called_with(action)
+            self.assertTrue(result['complete_prepare'])
+
+    def test_action_check_source_sync_complete(self, mock_logger):
```

Comment:
We're really not testing anything in this test. Only the bogus 'True' return value. We're not checking whether we're actually checking the last schema sync and updating the sync schema.
We're just mocking everything and check that we receive True as a result..

Reply:
Fixed.